.github/workflows/test.yml

```diff
@@ -10,19 +10,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-18.04, ubuntu-20.04]
+        os: [ubuntu-20.04, ubuntu-22.04]
         ruby: [2.7, '3.0', 3.1, 3.2]
+        redis-version: [6] # Redis 7 is failing (2023-01-17)
     services:
       redis:
-        image: redis
+        image: redis:${{ matrix.redis-version }}
```

Comment:
nachfolgende Frage, woher kommt die Matrix bzw. ihre Eingabe? ich finde das nirgendwo

Reply:
Das wird [in Zeile 15 angegeben](https://github.com/STUDITEMPS/predictor/blob/e28ad54b12b850521769950e721722ad679a7134/.github/workflows/test.yml#L15).

Reply:
achso danke dir